lib/ansible/plugins/callback/default.py

```diff
@@ -268,6 +268,22 @@ def v2_playbook_on_stats(self, stats):
 
         self._display.display("", screen_only=True)
 
+        # print custom stats
+        if C.SHOW_CUSTOM_STATS and stats.custom:
+            self._display.banner("CUSTOM STATS: ")
+            # per host
+            #TODO: come up with 'pretty format'
+            for k in sorted(stats.custom.keys()):
```

Comment:
Could use some docs/comments explaining what the stats.custom data structure looks like.